tvdbapiv2/rest.py

```diff
@@ -58,8 +56,10 @@ def request(self, method, url, query_params=None, headers=None, body=None, post_
         except RequestException as error:
             status = 0
             msg = "{0}\n{1}".format(type(error).__name__, text_type(error))
-            with suppress(AttributeError):
+            try:
```

Comment:
Why you prefer this?

Reply:
Because you added a dependency just for one `suppress` call...

Reply:
Mm okay